components/range.js

```diff
@@ -0,0 +1,62 @@
+const template = `
+<div class="c-range" :class="classes">
+  <label class="c-range__label" :for="name">
+    <span dir="ltr">{{label}}</span>
+  </label>
+  <small v-if="hint" class="c-range__hint">
+    <span dir="ltr">
+      {{ hint }}
+    </span>
+  </small>
+  <slot v-else name="hint"></slot>
+  <input class="c-range__input" :id="name" type="range" style="background-size: 50%;">
```

Comment:
Any time we use inline styling, can we use style bindings instead?

https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax-1